scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/signal/SignalApiImpl.java

```diff
@@ -66,8 +66,7 @@ public SignalApiImpl(String originator, TaskId taskId, SynchronizationInternal s
     private void init() throws SignalApiException {
         if (!isInitialized) {
             try {
-                // Initialize synchronization signals channel
-                synchronization.createChannelIfAbsent(SIGNALS_CHANNEL, true);
+                // Initialize job signals
```

Comment:
@fviale 
I removed the instruction **synchronization.createChannelIfAbsent(SIGNALS_CHANNEL, true);** because it is useless.
The signals channel is automatically created in the class **SchedulerFrontend**  in the method **initActivity()** -> **startSynchronizationService()**. Thus, avoiding this instruction each time the signal API is used.

Reply:
ok